inat.label.py

```
@@ -1024,7 +1024,7 @@ def format_scientific_name(observation_data):
     # Construct: italicize genus and epithet, not the rank label
     return f"__ITALIC_START__{genus}__ITALIC_END__ {rank_label[rank]} __ITALIC_START__{scientific_name}__ITALIC_END__"
 
-def create_inaturalist_label(observation_data, iconic_taxon_name, rtf_mode=False, show_common_names=False):
+def create_inaturalist_label(observation_data, iconic_taxon_name, rtf_mode=False, show_common_names=False, omit_notes=False, debug=False):
```

Ruff (0.14.5)

Unused function argument: `rtf_mode`

(ARG001)